examples/appSpec/zomes/three/code/index.ts

```diff
@@ -18,6 +19,20 @@ function test_debug(val: string): void {
   debug(val);
 }
 
+@serializable
+class TestClass {
```

Comment:
Is there a simpler way to define a class, and provide its types? without constructor?

Reply:
I just found out it is possible to do object literals with nested objects in AS so I made a more interesting test case
```
@can_stringify
class X {
  a: string
  b: i32
  c: Array<Y>
}

@can_stringify
class Y {
  n: bool
}

@zome_function
function test_debug_object(): void {
  let c: X = {a: "hi", b: 20, c: [{n: false},{n:true}]}
  debug(c);
}
```